charts/em-npa/Chart.yaml

```diff
@@ -1,11 +1,19 @@
 apiVersion: v2
 name: em-npa
 home: https://github.com/hmcts/em-native-pdf-annotator-app
-version: 1.0.19
+version: 2.0.0
```

Comment:
You will need to resolve merge conflict and update the chart version only to 1.0.21. Please refer to https://github.com/hmcts/em-native-pdf-annotator-app/pull/1395/files